app/assets/stylesheets/layout/common/_contact_panels.scss

```diff
@@ -1,6 +1,16 @@
+.l-contact-panels {
+  margin-top: 40px;
+  border-top: $default-border-thin;
+}
+
 .l-contact-panels--no-border-top {
```

Comment:
This is a little presentational and quite specific to `border-top`. What about `.l-contact-panels--flush`? Only a nitpick so feel free to ignore if you disagree.